content/posts/customizing/add-section.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "How to add a new section"
-date: 2020-06-07T06:20:50+06:00
+date: 2020-10-23T00:00:00+00:00
```

Comment:
I'm thinking i should change this back, i'm realizing its making the posts show up first where previously they all had the same date/time
![image](https://user-images.githubusercontent.com/8029578/97780122-02ec1e00-1b59-11eb-98f1-0cfd748a2cdb.png)